<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellWidget</class>
 <widget class="QWidget" name="CellWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>299</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="displayButton">
     <property name="text">
      <string>Cell Geometry</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,1,0">
      <item>
       <layout class="QHBoxLayout" name="cellFmtLayout">
        <item>
         <widget class="QLabel" name="cellEnabledLabel">
          <property name="text">
           <string>Enabled</string>
          </property>
          <property name="buddy">
           <cstring>cellEnabledBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cellEnabledBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="cellHorSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="cellScaleLabel">
          <property name="text">
           <string>Scale coordinates with cell</string>
          </property>
          <property name="buddy">
           <cstring>cellScaleBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cellScaleBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="cellEditLayout">
        <item>
         <widget class="QLabel" name="cellDimLabel">
          <property name="text">
           <string>Lattice constant:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="cellDimSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cellDimBox">
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cellFmt">
          <item>
           <property name="text">
            <string>Angstrom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bohr</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="cellVecLabel">
        <property name="text">
         <string>Cell vectors:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="cellVecTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>130</height>
         </size>
        </property>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>x</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>y</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>z</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cellTrajecButton">
        <property name="text">
         <string>Apply to trajectory</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
